### DISCUSSION TITLE
What does b17 actually stand for?

### DISCUSSION THREAD

**Discussion by Ge0** [Chia-Network/chia-blockchain#3702] [General]
Hello there,

I am reading chia's source code (especially chiapos) and I have come accross the `b17` symbol but I cannot understand what it does refer to. It's part of the plotting process but have no clue where it comes from.

Any idea?

**Comment by lin7sh** [comment]
I think it's deprecated version used it Beta17

**Reply by Ge0** [reply]
Thank you.
